h2o-r/scripts/h2o-r-test-setup.R

```diff
@@ -182,10 +182,6 @@ function() {
     cat(sprintf("[%s] %s\n", Sys.time(), paste0("Connect to h2o on IP: ",H2O.IP,", PORT: ",H2O.PORT)))
     h2o.init(ip = H2O.IP, port = H2O.PORT, startH2O = FALSE, strict_version_check = strict_version_check)
 
-    if (!is.null(RESULTS.DIR)) {
```

Comment:
Can we disable this with an option instead of deleting it completely? A note why is it disabled would be perhaps a good idea.

Also, we could leave it on for some smaller tests that run fast.

Reply:
Changed in following way: disabled by default, but you can specify the --restLog option, which will enable REST API logging.